### PR TITLE
Build linux packages files and push to respective repos

### PR DESCRIPTION
**Pull request by Ninja243**
Creates github action to build linux packages files and push them to the repos here: 
 - https://github.com/iits-consulting/ppa
 - https://github.com/iits-consulting/apk-repo
 - https://github.com/iits-consulting/rpm-repo

Closes https://github.com/iits-consulting/otc-auth/issues/25

### CODE REVIEW

**Comment by Ninja243** [comment]
PR to homebrew: https://github.com/Homebrew/homebrew-core/pull/136413

File: .goreleaser.yaml [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+
+nfpms:
+  - id: foo
+    package_name: otc-auth
+    file_name_template: "{{ .ConventionalFileName }}"
+    vendor: "iits-consulting"
+    homepage: "https://github.com/iits-consulting/otc-auth"
+    maintainer: "kontakt@iits-consulting.de"
+    description: "Test"
```
Comment (zeljkobekcic) [inline .goreleaser.yaml:55]: Maybe we should change just this here

**Review by zeljkobekcic — DISMISSED** [review]
There is just one "Test" in the .goreleaser.yaml. I am ATM not sure what it means

**Comment by Ninja243** [comment]
> There is just one "Test" in the .goreleaser.yaml. I am ATM not sure what it means

Good spot, I missed that. It refers to the app's description: https://goreleaser.com/customization/nfpm

File: Formula/otc-auth.rb [hunk 2]
```diff
@@ -0,0 +1,26 @@
+# Adds the otc-auth project to homebrew
+class OtcAuth < Formula
+  desc "Open Source CLI for the Open Telekom Cloud written in go"
+  homepage "https://github.com/iits-consulting/otc-auth"
+
+  url "https://github.com/iits-consulting/otc-auth.git",
+      tag:      "v2.0.0",
+      revision: "86b76b04813ce94cfaacd95f8653f2fe13851a60"
+  license "MIT"
```
Comment (zeljkobekcic) [inline Formula/otc-auth.rb:9]: Just make it GPLv3 and we are good to go!
https://github.com/iits-consulting/otc-auth/blob/main/LICENSE

**Review by zeljkobekcic — CHANGES_REQUESTED** [review]

**Review by zeljkobekcic — APPROVED** [review]

**Pull request merged**